features/pages/location_page.rb

```diff
@@ -0,0 +1,10 @@
+module Pages
+  class Location < SitePrism::Page
+    set_url '/locations/{id}'
```

Comment:
No. This is building on what is [already in place with `Pages::Locations`](https://github.com/guidance-guarantee-programme/pension_guidance/blob/37fae3cf14fdeb87098c8fb270efe1e0704c0ab4/features/pages/locations_page.rb#L3).